libc/config/linux/aarch64/entrypoints.txt

```diff
@@ -378,6 +378,9 @@ set(TARGET_LIBM_ENTRYPOINTS
     libc.src.math.canonicalize
     libc.src.math.canonicalizef
     libc.src.math.canonicalizel
+    libc.src.math.iscanonical
+    libc.src.math.iscanonicalf
+    libc.src.math.iscanonicall
     libc.src.math.cbrt
```

Comment:
Nit: should sort the entrypoint list with the `sort` command. Same for the other entrypoints.txt files.